server/tests/plugin/auror_testssl/test_parser.py

```diff
@@ -0,0 +1,165 @@
+# This file is part of sner4 project governed by MIT license, see the LICENSE.txt file.
+"""
+auror_testssl output parser tests
+"""
+
+import json
+import pytest
+from unittest.mock import patch
+from sner.plugin.auror_testssl.parser import ParserModule
+
+
+@pytest.fixture
+def dummy_pidb():
```

Comment:
question: what is the purpose here ? cannot we use original PIDB implementation ?